packages/react/src/AnchorNav/AnchorNav.features.stories.tsx

```diff
@@ -0,0 +1,220 @@
+      name: 'Data',
+      description: 'Test data',
+      control: {
+        type: 'object'
+      },
+      table: {
+        category: 'Story customization'
+      }
+    },
+    offset: {
+      name: 'Offset',
+      description: 'Offset from the top of the page',
+      control: {
+        type: 'number'
+      },
+      table: {
+        category: 'Story customization'
+      }
+    }
+  },
+  enableDefaultBgColor: {
+    description: 'Enable default background color',
+    control: {type: 'boolean'},
+    defaultValue: true,
+    table: {
+      category: 'AnchorNav'
+    }
+  }
+} as ComponentMeta<typeof AnchorNav>
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const Template: ComponentStory<typeof AnchorNav> = (_, storyArgs: any) => {
+  const storyData = Object.entries(storyArgs.args.data) as [string, string][]
+  return (
+    <div style={{backgroundColor: 'var(--brand-color-canvas-default)'}}>
+      <RedlineBackground height={storyArgs.args.offset}>
+        <Stack direction="vertical" gap="none" justifyContent="center" style={{textAlign: 'center'}}>
+          <Heading as="h1" size="6">
+            Offset {storyArgs.args.offset}px from top
+          </Heading>
+          <Text as="p" variant="muted" size="200">
+            (configure me using Storybook Controls)
+          </Text>
+        </Stack>
+      </RedlineBackground>
+      <AnchorNav {...storyArgs.args}>
+        {storyData.map(([key, value]) => (
+          <AnchorNav.Link href={value} key={value}>
+            {key}
+          </AnchorNav.Link>
+        ))}
+        <AnchorNav.Action href="#">Sign up</AnchorNav.Action>
+      </AnchorNav>
+      {/**
+       *  The following markup is provided for demonstration purposes only.
+       *  It is not part of the AnchorNav component.
+       */}
+      <Stack
+        direction="vertical"
+        justifyContent="space-around"
+        gap="none"
+        style={{marginBottom: '100px'}}
+        padding="none"
+      >
+        {storyData.map(([key, value]) => (
+          <RedlineBackground key={key}>
+            <Stack
+              key={value}
+              id={value}
+              direction="vertical"
+              style={{
+                padding: '500px var(--base-size-24)'
+              }}
+            >
+              <Heading>{key}</Heading>
+              <Text as="p">
+                AnchorNav is a component that allows users to navigate to different sections of a page.
+              </Text>
+            </Stack>
+          </RedlineBackground>
+        ))}
+      </Stack>
+    </div>
+  )
+}
+
+export const FewerAnchorLinks = Template.bind({})
+FewerAnchorLinks.storyName = 'Fewer anchor links'
+FewerAnchorLinks.args = {
+  data: {
+    ['GitHub vs Jenkins']: 'githubvsjenkins',
+    ['GitHub vs GitLab']: 'githubvsgitlab',
+    ['GitHub vs CircleCI']: 'githubvscircleci'
+  }
+} as never
+
+export const CustomBackground = ({data, ...args}: {data: MockData; offset: number}) => {
```

Comment:
This story throws me off a bit because it isn't directly related to the component. Is the background for the navigation always transparent? Is there control for the menu transparency?

Reply:
Yes, navigation is transparent by default per design review feedback. There is an optional prop to enable a background, which I'm using in the stories. There's a weird canvas mode bug in Storybook, where the background doesn't appear correctly unless you're in iframe mode 🤷